module-system/sov-modules-macros/src/module_info.rs

```diff
@@ -249,6 +254,19 @@ fn make_fn_address(
     }
 }
 
+fn make_fn_dependencies(modules: Vec<&proc_macro2::Ident>) -> proc_macro2::TokenStream {
+    let address_tokens = modules.iter().map(|ident| {
+        quote::quote! {
+            &self.#ident.address()
+        }
+    });
+
+    quote::quote! {
+        fn dependencies(&self) -> Vec<&<Self::Context as sov_modules_api::Spec>::Address> {
```

Comment:
We need a fully qualified name for `Vec` here (this will prevent name collisions if someone defines `Vec` struct in the file where the macro is used)